src/lib/viewers/doc/docAssets.js

```diff
@@ -6,5 +6,6 @@ export const IMAGES = `${STATIC_URI}/images/`;
 export const WORKER = `${STATIC_URI}/pdf.worker.min.mjs`;
 
 export const JS = [`${STATIC_URI}/pdf.min.mjs`, `${STATIC_URI}/pdf_viewer.min.mjs`, `${STATIC_URI}/exif.min.js`];
+export const EXIF = [`${STATIC_URI}/exif.min.js`];
```

Comment:
Has this been deployed to all the CDNs?

Reply:
No these are actually included in the preview asset that we deploy to the CDN.

Reply:
Probably makes sense to either have a variable for the `EXIF` string or import it as part of the `JS` array

Reply:
This will completely change in a subsequent PR because the exif library doesn't work for webp files.